troubleshooting/log-bundle.md

```diff
@@ -0,0 +1,70 @@
+
+## Data included in log bundle
+
+Manifests
+Systemd logs from bootstrap / masters
+Container inspect / logs
+
+## Where to start reading a log bundle
+
+### Payload and installer version
+First its crucial to ensure the installer and release payload versions match:
+```
+$ oc adm release info registry.ci.openshift.org/origin/release:4.8 --commits | grep installer
+  baremetal-installer                            https://github.com/vrutkovs/installer                                       4b15f0537ad76206c8ba9ba042b404f8c00a295a
+  installer                                      https://github.com/vrutkovs/installer                                       4b15f0537ad76206c8ba9ba042b404f8c00a295a
+  installer-artifacts                            https://github.com/vrutkovs/installer                                       4b15f0537ad76206c8ba9ba042b404f8c00a295a
+```
+This command returns the expected commit used to build an installer.
+
+When the cluster is being installed, installer is recording information about itself in the ConfigMap:
+```
+$ cat rendered-assets/openshift/openshift/openshift-install-manifests.yaml
+apiVersion: v1
+data:
+  invoker: openshift-internal-ci/release-openshift-okd-installer-e2e-aws-upgrade/1396004378611027968
+  version: unreleased-master-4619-g4b15f0537ad76206c8ba9ba042b404f8c00a295a
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: openshift-install-manifests
+  namespace: openshift-config
+```
+
+`invoker` format is `unreleased-<branch>-<number of commits>-g<commit hash>`. As you can see,
+commit hashes are matching, so a valid installer was used to run this cluster.
+
+### Release image
+
+To verify that a correct release payload was used check `bootstrap/journals/release-image.log`:
+```
+-- Journal begins at Sat 2021-05-22 07:32:02 UTC, ends at Sat 2021-05-22 08:06:41 UTC. --
+May 22 07:34:45 ip-10-0-6-51 systemd[1]: Starting Download the OpenShift Release Image...
+May 22 07:34:45 ip-10-0-6-51 release-image-download.sh[762]: Pulling registry.build01.ci.openshift.org/ci-op-fggt4c0x/release@sha256:c843adfb4bddd22f7256325ed387d9a685be451ca6d708a56ecb8e8a738b7892...
+May 22 07:34:46 ip-10-0-6-51 podman[813]: 2021-05-22 07:34:46.474241096 +0000 UTC m=+0.476275595 system refresh
+May 22 07:34:46 ip-10-0-6-51 podman[813]: 2021-05-22 07:34:46.807049267 +0000 UTC m=+0.809083790 image pull  
+May 22 07:34:46 ip-10-0-6-51 release-image-download.sh[813]: cfd14e030756425d5f7b09f44926c50c87256f5dd75303f845c27e22790d423e
+May 22 07:34:46 ip-10-0-6-51 systemd[1]: Finished Download the OpenShift Release Image.
+```
+
+It should be inspectable with `oc adm release info <payload pullspec>`.
+
+After release image has been pulled, bootstrap node extracts Machine Config Daemon and updates plain 
+FCOS to OKD content in `release-image-download.service` by pulling `machine-os-content` image from 
+the payload, extracting ostree commit and pivoting bootstrap into it.
```

Comment:
```suggestion
After a release image has been pulled by `release-image-download.service`, the bootstrap node extracts
the Machine Config Daemon (MCD) from it and updates the plain Fedora CoreOS (FCOS) it has booted
from to a version adjusted for the specific version of OKD. The service deploys it on the bootstrap node by
using `podman` to pull the `machine-os-content` image specified in the payload from the referenced registry, extracting the OSTree commit embedded in the image, and re-booting the OS into the desired version (this
is also called _pivoting_).
```

Reply:
Perhaps we should describe how OKD boot differs from OCP boot - pivoting bootstrap / nodes - and then proceed to troubleshooting instructions?